src/cdi_CPEloss/test/001-H-001

```diff
@@ -0,0 +1,72 @@
+ -5.213922E+01 -5.171510E+01 -5.141800E+01 -5.118437E+01  0.000000E+00  0.000000E+00
+ -5.318258E+01 -5.228064E+01 -5.180523E+01 -5.147755E+01  0.000000E+00  0.000000E+00
+ -5.637066E+01 -5.357591E+01 -5.244426E+01 -5.189350E+01  0.000000E+00  0.000000E+00
+ -6.092108E+01 -5.725646E+01 -5.408364E+01 -5.260964E+01  0.000000E+00  0.000000E+00
+ -5.580772E+01 -5.561951E+01 -5.546118E+01 -5.532450E+01 -5.520374E+01 -5.508168E+01
+ -5.610727E+01 -5.586087E+01 -5.566335E+01 -5.549845E+01 -5.535631E+01 -5.521549E+01
+ -5.653690E+01 -5.617968E+01 -5.591702E+01 -5.570917E+01 -5.553641E+01 -5.536999E+01
+ -5.730165E+01 -5.665041E+01 -5.625775E+01 -5.597647E+01 -5.575623E+01 -5.555279E+01
+ -5.953744E+01 -5.755491E+01 -5.677840E+01 -5.634236E+01 -5.603834E+01 -5.577650E+01
+ -6.018269E+01 -6.003039E+01 -5.989826E+01 -5.978156E+01 -5.967707E+01 -5.958229E+01
+ -6.041289E+01 -6.022468E+01 -6.006635E+01 -5.992969E+01 -5.980948E+01 -5.970198E+01
+ -6.071244E+01 -6.046604E+01 -6.026852E+01 -6.010364E+01 -5.996214E+01 -5.983797E+01
+ -6.114207E+01 -6.078485E+01 -6.052219E+01 -6.031436E+01 -6.014238E+01 -5.999541E+01
+ -6.190682E+01 -6.125558E+01 -6.086292E+01 -6.058167E+01 -6.036240E+01 -6.018235E+01
+          1002
+             1  2.014102E+00  3.344042E-24
+ -4.907260E+01 -4.827732E+01 -4.782467E+01  0.000000E+00  0.000000E+00  0.000000E+00
+ -5.192024E+01 -4.940434E+01 -4.840673E+01  0.000000E+00  0.000000E+00  0.000000E+00
+ -5.643790E+01 -5.278527E+01 -4.977505E+01  0.000000E+00  0.000000E+00  0.000000E+00
+ -6.103644E+01 -5.734334E+01 -5.352644E+01  0.000000E+00  0.000000E+00  0.000000E+00
+ -6.553815E+01 -6.182302E+01 -5.787268E+01  0.000000E+00  0.000000E+00  0.000000E+00
+ -5.227664E+01 -5.201184E+01 -5.180262E+01 -5.162778E+01  0.000000E+00  0.000000E+00
+ -5.275293E+01 -5.235545E+01 -5.207168E+01 -5.184850E+01  0.000000E+00  0.000000E+00
+ -5.367779E+01 -5.288295E+01 -5.244089E+01 -5.213212E+01  0.000000E+00  0.000000E+00
+ -5.652548E+01 -5.401083E+01 -5.303194E+01 -5.252930E+01  0.000000E+00  0.000000E+00
+ -6.104251E+01 -5.739373E+01 -5.444447E+01 -5.319447E+01  0.000000E+00  0.000000E+00
+ -5.646882E+01 -5.628600E+01 -5.613149E+01 -5.599769E+01 -5.587942E+01 -5.576556E+01
+ -5.675750E+01 -5.652027E+01 -5.632867E+01 -5.616793E+01 -5.602918E+01 -5.589808E+01
+ -5.716494E+01 -5.682677E+01 -5.657451E+01 -5.637323E+01 -5.620537E+01 -5.605087E+01
+ -5.786180E+01 -5.727095E+01 -5.690121E+01 -5.663185E+01 -5.641940E+01 -5.623129E+01
+ -5.978558E+01 -5.808307E+01 -5.738950E+01 -5.698160E+01 -5.669204E+01 -5.645151E+01
+ -6.085025E+01 -6.070150E+01 -6.057204E+01 -6.045743E+01 -6.035462E+01 -6.026131E+01
+ -6.107399E+01 -6.089117E+01 -6.073666E+01 -6.060287E+01 -6.048487E+01 -6.037925E+01
+ -6.136267E+01 -6.112544E+01 -6.093384E+01 -6.077311E+01 -6.063467E+01 -6.051298E+01
+ -6.177011E+01 -6.143194E+01 -6.117968E+01 -6.097841E+01 -6.081093E+01 -6.066739E+01
+ -6.246697E+01 -6.187612E+01 -6.150638E+01 -6.123703E+01 -6.102505E+01 -6.085008E+01
```

Comment:
I think it would be helpful to have a `README` file in the test directory that describes how this data was generated and clarifies that is only useful for testing this Draco package.

Reply:
Done.